p2p-specs/p2p-interface.md

```diff
@@ -572,9 +573,19 @@ The ENR MAY contain the following entries:
 
 Specifications of these parameters can be found in the [ENR Specification](http://eips.ethereum.org/EIPS/eip-778).
 
+#### Chain id field
+
+ENRs MUST carry a `chain_id` key containing the ID of the network the bundler is connected to. This is done to ensure connections are made with peers on the intended Ethereum network.
+
+| Key        | Value        |
+|:-----------|:-------------|
+| `chain_id` | SSZ `uint64` |
+
+Clients SHOULD connect to peers with a `chain_id` that matches local values.
+
 #### Mempools bitfield
 
-The ENR `mempools` entry signifies the mempools subnet bitfield with the following form
+The ENR `mempool_subnets` entry signifies the mempools subnet bitfield with the following form
```

Comment:
There is no definition what a "subnet" is. I thought it was defined as mempool_ID modulu 1024.

Reply:
I think we can remove mempool subnets entirely. They don't have a meaning at the moment.